DigitalLearningSolutions.Web.Tests/Controllers/LearningMenu/LearningMenuControllerTests.cs

```diff
@@ -85,5 +85,38 @@ public void ContentViewer_should_render_view()
             result.Should().BeViewResult()
                 .Model.Should().BeEquivalentTo(expectedModel);
         }
+
+        [Test]
+        public void Index_valid_customisation_id_should_call_the_course_content_service()
+        {
+            // Given
+            const int progressId = 13;
+            var defaultCourseContent = CourseContentHelper.CreateDefaultCourseContent(CustomisationId);
+            A.CallTo(() => courseContentService.GetCourseContent(CustomisationId)).Returns(defaultCourseContent);
+            A.CallTo(() => courseContentService.GetProgressId(CandidateId, CustomisationId)).Returns(progressId);
+
+            // When
+            controller.Index(CustomisationId);
+
+            // Then
+            A.CallTo(() => courseContentService.GetCourseContent(CustomisationId)).MustHaveHappened();

```

Comment:
This assertion isn't necessary as it isn't relevant to what's being tested. I'd suggest testing that this is called in a separate test 🙂